### PR TITLE
Update developmentEnvironment.md

### PR DESCRIPTION
**Pull request by peterkolcza**
Modification of the windows install section

### CODE REVIEW

**Comment by szerintedmi** [comment]
Thank you!
Did you need to install truffle globally? It's a dev dependency in `packages.json` and it should be installed locally when you run `npm install`. 
Then you should be able to run truffle via npm scripts (see in `packages.json`, scripts are referencing to the local `node_modules` install) or using `$(npm bin)/truffle`
If you are in contract development then you can choose to install globally (so you have to convenience of just entering `truffle compile` and test w/ different truffle versions etc. 

This repo is submodule of `augmint-web` too and would be good to save the hassle for FE devs to install truffle globally only to be able to test FE locally (i.e. FE repo has scripts calling the `augmint-contracts` submodule's scripts, eg. `npm run ganache:runmigrate`).

All this hassle because it would make it easier to ensure everyone uses the right truffle version ( truffle upgrades sometimes require tweaks which causing older truffle versions not to work anymore)

@reyrodrigez : you just mentioned that you had issues with the relative `node_modules` paths in `augmint-web` - feel free to jump in with your thoughts, there might be a simpler/better way to tackle this

File: docs/developmentEnvironment.md [hunk 1]
```diff
@@ -23,17 +23,30 @@ _NB: these steps are likely to work on linux too but it's not tested yet_
 
 ### Windows
 
-_NB: windows install was not tested since a while, update on it is welcome_
+_Note: It is recommended to use PowerShell (win+X => powershell)_
```
Comment (reyrodrigez) [inline docs/developmentEnvironment.md]: 👍

File: docs/developmentEnvironment.md [hunk 2]
```diff
@@ -23,17 +23,30 @@ _NB: these steps are likely to work on linux too but it's not tested yet_
...
+   npm install -g ganache-cli
+   ```
+   _Config details in runganche.bat_
+   
+1. Get the source code:
     ```
-    nvm install 8.5.0
-    nvm use 8.5.0
-    git clone https://github.com/Augmint/augmint-contracts.git
+    git clone https://github.com/Augmint/augmint-web.git
```
Comment (reyrodrigez) [inline docs/developmentEnvironment.md]: wait, I'm a bit confused here. you clone the `augmint-web` repo then cd to `augmint-contracts`

Comment (coolczus) [reply docs/developmentEnvironment.md]: Yes, this must be augmint-contracts.

**Comment by coolczus** [comment]
@szerintedmi, unfortunately it only worked when I have installed it globally.
 I have checked packages.json, but the ganache:runmigrate have not worked under windows:

```
> npm run ganache:runmigrate

> augmint-contracts@0.1.0 ganache:runmigrate C:\Users\KP\Desktop\tmp\augmint-contracts
> npm start

> augmint-contracts@0.1.0 start C:\Users\KP\Desktop\tmp\augmint-contracts
> ./runganache.sh & $(npm bin)/truffle migrate && echo 'Migration done. Contracts deployed to ganache. Contract artifacts are in build/contracts folder.' & wait

'.' is not recognized as an internal or external command,
operable program or batch file.
'$' is not recognized as an internal or external command,
```

But it should work locally long term. I think I give it another try.

**Comment by szerintedmi** [comment]
there is a `runganache.bat` for windows (not tested for ages)

**Comment by szerintedmi** [comment]
This PR became obsolete

**Pull request closed without merging**